test/living-dom/non-document-type-child-node.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+const load = require("../util").load(__dirname);
+
+exports["Element should implement NonDocumentTypeChildNode:nextElementSibling"] = function (t) {
```

Comment:
Will need tests for CharacterData too (just testing `Text` and `Comment` is probably fine; no need to test `ProcessingInstruction`).